getopt.c:457: warning: incompatible implicit declaration of built-in function 'strlen'

--- src/getopt.c.orig	Fri Jun 19 14:38:04 2015
+++ src/getopt.c	Fri Jun 19 14:38:27 2015
@@ -30,6 +30,8 @@
 #include "config.h"
 #endif
 
+#include <string.h>
+
 #ifdef __GNUC__
 #define alloca __builtin_alloca
 #else /* not __GNUC__ */
